pallets/farming/src/lib.rs

```diff
@@ -388,6 +398,29 @@ pub mod pallet {
     #[pallet::storage]
     pub type PoolFarmers<T: Config> =
         StorageMap<_, Identity, T::AccountId, Vec<PoolFarmer<T>>, ValueQuery>;
+
+    #[pallet::storage]
+    #[pallet::getter(fn lp_min_xor_for_bonus_reward)]
+    pub type LpMinXorForBonusReward<T: Config> = StorageValue<_, Balance, ValueQuery>;
```

Comment:
You have to add `DefaultLpMinXorForBonusReward` value

Reply:
And, probably add the migration to inroduce the first value. @vovac12 what do you think?
And what about adding the default value in the chain spec?

Reply:
It's enough to add default value